integration_test_lite.py

```diff
@@ -67,7 +67,7 @@ def basic_request(
 
         return session.request(method, url, params=params, headers=headers)
 
-    async def bad_response(self, endpoint):
+    async def test_endpoint(self, endpoint):
         """Tests an endpoint for an unexpected response
```

Comment:
Is it still only test for unexpected responses?

Reply:
This function actually tests all responses. I'll update the docstring